app/views/pages/home.html.erb

```diff
@@ -14,20 +14,18 @@
         Créez un carnet de bord digital pour votre nouveau petit héro !</p>
         <br>
```

Comment:
Why 'br' tag  ? Is it not preferable with margin / paddin bottom ?

Reply:
I'll fix it ASAP, it's an easy way and it's more quickly, that's why. But it's a bit painfull yea.

Reply:
Completely understand. Will check everything together. (must be cool)